apmpackage/apm/0.1.0/data_stream/internal_metrics/fields/fields.yml

```diff
@@ -153,8 +163,11 @@
     Transactions that are 'sampled' will include all available information. Transactions that are not sampled will not have spans or context.
 - name: transaction.self_time.count
   type: long
+  description: Number of aggregated transactions.
 - name: transaction.self_time.sum.us
   type: long
+  description: |
+    Aggregated transaction durations, excluding the durations where a direct child was running, in microseconds.
```

Comment:
nit: in https://github.com/elastic/apm-server/pull/4689/files#diff-8d8bd845e531a70d1d9c1559419ef257df67122295376080d90f3610442764d1R148 you use singular, here plural for `durations`. Would you mind aligning?

Reply:
Done, made them singular. I did change the second "durations" to "time periods", to clarify that the time periods are being excluded from the aggregated duration.